com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/util/ReportGenerator.java

```diff
@@ -91,7 +91,10 @@ public ReportGenerator(Project project) throws IOException {
         this.excludedResources = new HashMap<String, ResourceEntry>();
 
         if (project.option("archive", "false").equals("true")) {
-            parseArchiveBuilder(project.getArchiveBuilder());
+            ArchiveBuilder archiveBuilder = project.getArchiveBuilder();
+            if (archiveBuilder != null) {
+                parseArchiveBuilder(archiveBuilder);
```

Comment:
similiar to archive check. if we trigger two builds without any changes. We can't produce report in such case